src/components/Dashboard/MyToken/MyToken.jsx

```diff
@@ -375,9 +404,12 @@ class MyTokens extends Component {
 
 
 
-          <Grid item xs={6} sm={6}>
+          <Grid item xs={12} sm={12} md={6}>
```

Comment:
the values apply for the specified size and all larger values, so you can shorten this to
`<Grid item xs={12} md={6}>`

since `md` is the breakpoint when you want the grid item to be smaller, `sm` will implicitly have the same value as `xs`